packages/repository/test/unit/decorator/relation.decorator.unit.ts

```diff
@@ -109,6 +118,85 @@ describe('relation decorator', () => {
       });
     });
   });
+
+  describe('belongsTo', () => {
+    it('creates juggler property metadata', () => {
+      @model()
+      class AddressBook extends Entity {
+        @property({id: true})
+        id: number;
+      }
+      class Address extends Entity {
+        @belongsTo(() => AddressBook)
+        addressBookId: number;
+      }
+      const jugglerMeta = MetadataInspector.getAllPropertyMetadata(
+        MODEL_PROPERTIES_KEY,
+        Address.prototype,
+      );
+      expect(jugglerMeta).to.eql({
+        addressBookId: {
+          type: Number,
+          required: true,
```

Comment:
Is `required: true` set by juggler because this is the only property in the model def?

Reply:
No, `required: true` used to be set by `@belongsTo` decorator before I changed this behavior based on our discussion.